

Started GET "/" for 127.0.0.1 at 2015-03-20 15:08:55 +0530
Connecting to database specified by database.yml

Mysql2::Error (Unknown database 'levi_assignment_development'):
  mysql2 (0.3.18) lib/mysql2/client.rb:70:in `connect'
  mysql2 (0.3.18) lib/mysql2/client.rb:70:in `initialize'
  activerecord (3.2.18) lib/active_record/connection_adapters/mysql2_adapter.rb:16:in `new'
  activerecord (3.2.18) lib/active_record/connection_adapters/mysql2_adapter.rb:16:in `mysql2_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:315:in `new_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:325:in `checkout_new_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:247:in `block (2 levels) in checkout'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:242:in `loop'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:242:in `block in checkout'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:239:in `checkout'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:102:in `block in connection'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:101:in `connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:410:in `retrieve_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_specification.rb:171:in `retrieve_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_specification.rb:145:in `connection'
  activerecord (3.2.18) lib/active_record/query_cache.rb:67:in `rescue in call'
  activerecord (3.2.18) lib/active_record/query_cache.rb:61:in `call'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:479:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.18) lib/active_support/callbacks.rb:405:in `_run__1009186584__call__909005537__callbacks'
  activesupport (3.2.18) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.18) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.18) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.18) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.18) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.18) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.18) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.18) lib/rails/engine.rb:484:in `call'
  railties (3.2.18) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.18) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:138:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:94:in `run'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/server.rb:191:in `block in start_thread'


  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/_trace.erb (0.9ms)
  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/_request_and_response.erb (1.8ms)
  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/diagnostics.erb within rescues/layout (8.9ms)


Started GET "/" for 127.0.0.1 at 2015-03-20 15:10:11 +0530

Mysql2::Error (Unknown database 'levi_assignment_development'):
  mysql2 (0.3.18) lib/mysql2/client.rb:70:in `connect'
  mysql2 (0.3.18) lib/mysql2/client.rb:70:in `initialize'
  activerecord (3.2.18) lib/active_record/connection_adapters/mysql2_adapter.rb:16:in `new'
  activerecord (3.2.18) lib/active_record/connection_adapters/mysql2_adapter.rb:16:in `mysql2_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:315:in `new_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:325:in `checkout_new_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:247:in `block (2 levels) in checkout'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:242:in `loop'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:242:in `block in checkout'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:239:in `checkout'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:102:in `block in connection'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:101:in `connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:410:in `retrieve_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_specification.rb:171:in `retrieve_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_specification.rb:145:in `connection'
  activerecord (3.2.18) lib/active_record/query_cache.rb:67:in `rescue in call'
  activerecord (3.2.18) lib/active_record/query_cache.rb:61:in `call'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:479:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.18) lib/active_support/callbacks.rb:405:in `_run__1009186584__call__909005537__callbacks'
  activesupport (3.2.18) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.18) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.18) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.18) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.18) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.18) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.18) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.18) lib/rails/engine.rb:484:in `call'
  railties (3.2.18) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.18) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:138:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:94:in `run'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/server.rb:191:in `block in start_thread'


  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/_trace.erb (0.9ms)
  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/_request_and_response.erb (1.1ms)
  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/diagnostics.erb within rescues/layout (7.4ms)


Started GET "/index" for 127.0.0.1 at 2015-03-20 15:10:19 +0530

Mysql2::Error (Unknown database 'levi_assignment_development'):
  mysql2 (0.3.18) lib/mysql2/client.rb:70:in `connect'
  mysql2 (0.3.18) lib/mysql2/client.rb:70:in `initialize'
  activerecord (3.2.18) lib/active_record/connection_adapters/mysql2_adapter.rb:16:in `new'
  activerecord (3.2.18) lib/active_record/connection_adapters/mysql2_adapter.rb:16:in `mysql2_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:315:in `new_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:325:in `checkout_new_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:247:in `block (2 levels) in checkout'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:242:in `loop'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:242:in `block in checkout'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:239:in `checkout'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:102:in `block in connection'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:101:in `connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:410:in `retrieve_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_specification.rb:171:in `retrieve_connection'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_specification.rb:145:in `connection'
  activerecord (3.2.18) lib/active_record/query_cache.rb:67:in `rescue in call'
  activerecord (3.2.18) lib/active_record/query_cache.rb:61:in `call'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:479:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.18) lib/active_support/callbacks.rb:405:in `_run__1009186584__call__909005537__callbacks'
  activesupport (3.2.18) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.18) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.18) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.18) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.18) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.18) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.18) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.18) lib/rails/engine.rb:484:in `call'
  railties (3.2.18) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.18) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:138:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:94:in `run'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/server.rb:191:in `block in start_thread'


  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/_trace.erb (1.0ms)
  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/_request_and_response.erb (0.7ms)
  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/diagnostics.erb within rescues/layout (6.8ms)


Started GET "/index" for 127.0.0.1 at 2015-03-20 15:15:22 +0530
Connecting to database specified by database.yml

ActionController::RoutingError (No route matches [GET] "/index"):
  actionpack (3.2.18) lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.18) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.18) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.18) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.18) lib/rails/engine.rb:484:in `call'
  railties (3.2.18) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.18) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:138:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:94:in `run'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/server.rb:191:in `block in start_thread'


  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/routing_error.erb within rescues/layout (19.0ms)


Started GET "/index" for 127.0.0.1 at 2015-03-20 15:18:50 +0530

ActionController::RoutingError (No route matches [GET] "/index"):
  actionpack (3.2.18) lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.18) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.18) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.18) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.18) lib/rails/engine.rb:484:in `call'
  railties (3.2.18) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.18) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:138:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:94:in `run'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/server.rb:191:in `block in start_thread'


  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/routing_error.erb within rescues/layout (0.5ms)


Started GET "/" for 127.0.0.1 at 2015-03-20 15:19:38 +0530

ActionController::RoutingError (uninitialized constant WelcomeController):
  activesupport (3.2.18) lib/active_support/inflector/methods.rb:230:in `block in constantize'
  activesupport (3.2.18) lib/active_support/inflector/methods.rb:229:in `each'
  activesupport (3.2.18) lib/active_support/inflector/methods.rb:229:in `constantize'
  actionpack (3.2.18) lib/action_dispatch/routing/route_set.rb:69:in `controller_reference'
  actionpack (3.2.18) lib/action_dispatch/routing/route_set.rb:54:in `controller'
  actionpack (3.2.18) lib/action_dispatch/routing/route_set.rb:32:in `call'
  journey (1.0.4) lib/journey/router.rb:68:in `block in call'
  journey (1.0.4) lib/journey/router.rb:56:in `each'
  journey (1.0.4) lib/journey/router.rb:56:in `call'
  actionpack (3.2.18) lib/action_dispatch/routing/route_set.rb:608:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/best_standards_support.rb:17:in `call'
  rack (1.4.5) lib/rack/etag.rb:23:in `call'
  rack (1.4.5) lib/rack/conditionalget.rb:25:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/head.rb:14:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/params_parser.rb:21:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/flash.rb:242:in `call'
  rack (1.4.5) lib/rack/session/abstract/id.rb:210:in `context'
  rack (1.4.5) lib/rack/session/abstract/id.rb:205:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/cookies.rb:341:in `call'
  activerecord (3.2.18) lib/active_record/query_cache.rb:64:in `call'
  activerecord (3.2.18) lib/active_record/connection_adapters/abstract/connection_pool.rb:479:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.18) lib/active_support/callbacks.rb:405:in `_run__149995000__call__350008964__callbacks'
  activesupport (3.2.18) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.18) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.18) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.18) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.18) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.18) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.18) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.18) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.18) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.18) lib/rails/engine.rb:484:in `call'
  railties (3.2.18) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.18) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:138:in `service'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/httpserver.rb:94:in `run'
  /home/anchanto/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/webrick/server.rb:191:in `block in start_thread'


  Rendered /home/anchanto/.rvm/gems/ruby-1.9.3-p547@rails-3.2.17/gems/actionpack-3.2.18/lib/action_dispatch/middleware/templates/rescues/routing_error.erb within rescues/layout (0.7ms)


Started GET "/" for 127.0.0.1 at 2015-03-20 15:25:36 +0530
Connecting to database specified by database.yml
Processing by HomeController#index as HTML
  Rendered home/index.html.erb within layouts/application (1.9ms)
Warning. Error encountered while saving cache /home/anchanto/anjali_workspace/levi_assignment/tmp/cache/sass/2b5e02f9d4e102112aa0250e4a5cce945de46c92/home.css.scssc: can't dump anonymous class #<Class:0x9e41ef8>
Compiled home.css  (25ms)  (pid 7259)
Compiled application.css  (32ms)  (pid 7259)
Compiled jquery.js  (19ms)  (pid 7259)
Compiled jquery_ujs.js  (0ms)  (pid 7259)
Compiled home.js  (361ms)  (pid 7259)
Compiled application.js  (452ms)  (pid 7259)
Completed 200 OK in 537.0ms (Views: 536.7ms | ActiveRecord: 0.0ms)


Started GET "/assets/home.css?body=1" for 127.0.0.1 at 2015-03-20 15:25:37 +0530
Served asset /home.css - 200 OK (2ms)


Started GET "/assets/home.js?body=1" for 127.0.0.1 at 2015-03-20 15:25:37 +0530
Served asset /home.js - 200 OK (20ms)


Started GET "/assets/application.css?body=1" for 127.0.0.1 at 2015-03-20 15:25:37 +0530
Served asset /application.css - 200 OK (3ms)


Started GET "/assets/jquery.js?body=1" for 127.0.0.1 at 2015-03-20 15:25:37 +0530
Served asset /jquery.js - 200 OK (4ms)


Started GET "/assets/application.js?body=1" for 127.0.0.1 at 2015-03-20 15:25:37 +0530
Served asset /application.js - 200 OK (9ms)


Started GET "/assets/jquery_ujs.js?body=1" for 127.0.0.1 at 2015-03-20 15:25:37 +0530
Served asset /jquery_ujs.js - 200 OK (1ms)
